### DISCUSSION TITLE
"TBA" for Basingstoke & Deane Waste UK, Recycle and Garden

### DISCUSSION THREAD

**Discussion by clipzwb** [mampfes/hacs_waste_collection_schedule#1523] [Q&A]
Since last week these are all showing TBA. Garden has showed "TBA" for some months but thats probably because I havent renewed garden subscription with local council, but waste and recycling have been fine until last week where they both show TBA constantly now.

**Comment by 5ila5** [comment]
might be duplicate of #1475
which was fixed by #1477 which will be part of release 1.45.0 or is available now as master (git) version (HACS: 3-dot menu -> re-download -> change version to master (Will increase update cycles by a lot))

**Reply by clipzwb** [reply]
Thanks for this. I have now tried this but still says TBA after redownloading "master" and restarting HA. Tried this process a couple of times but still no luck

**Reply by clipzwb** [reply]
I have manually deleted and replaced the basingstoke .py in the custom sources folder, with the latest downloaded from master and restarted ha and all seems to be ok for now. thanks

**Comment by stormzak** [comment]
Suspect this error I have below is or could be the same as above.

fetch failed for source Basingstoke and Deane Borough Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/basingstoke_gov_uk.py", line 65, in fetch date=datetime.strptime(schedule.text, "%A, %d %B %Y").date(), ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/_strptime.py", line 568, in _strptime_datetime tt, fraction, gmtoff_fraction = _strptime(data_string, format) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/_strptime.py", line 352, in _strptime raise ValueError("unconverted data remains: %s" % ValueError: unconverted data remains: (Christmas and New Year rescheduling)

**Reply by 5ila5** [reply]
yes same error as in  #1475
